### ISSUE TITLE
ObjectStatus should have aria-label if it only has an icon

### ISSUE THREAD

**Issue by jbadan** [SAP/fundamental-styles#1217]
https://github.com/SAP/fundamental-styles/blob/master/stories/object-status/object-status.stories.js#L33

0.11.0-rc.86

**Comment by stefanoScalzo** [comment]
This has been fixed, only object status with ONLY Icon have aria-label defined
